### DISCUSSION TITLE
Search capability support is not present for select

### DISCUSSION THREAD

**Discussion by apnavik** [twbs/bootstrap#36695] [General] [answered]
I am not able to find capability to provide search capability for select tag.

please find below image for reference

![image](https://user-images.githubusercontent.com/17467883/177749254-876d61de-adb9-435d-b176-e1d463960946.png)

**Comment by florianlacreuse** [comment]
This is normal, Bootstrap does not offer such a component. You need to use a third party component.

**Comment by vanillajonathan** [comment, marked as answer]
As @florianlacreuse mentioned this is not a functionality included in Bootstrap. If you want such a functionality you can accomplish it with a third-party library such as [select2](https://select2.org/), [selectize.js](https://selectize.dev/) or [Tom Select](https://tom-select.js.org/). Tom Select includes themes for both Bootstrap 4 and 5.

**Reply by apnavik** [reply]
Thanks @florianlacreuse for information.
Thank you @vanillajonathan. Yes, Tom Select is good option which works with Bootstrap 5.
